finance/funding_requests.md

```diff
@@ -0,0 +1,52 @@
+
+Funding requests can be made to the Astropy Finance Committee.  The following provides a description for how to obtain funding for different types of activities. 
+
+
+## Travel
+
+1. Send an email to astropy-finance-committee@googlegroups.com with the following information:
+* Name
+* Description
+* Total Request
+* Itemized Budget
+
+Costs for per diem can use the [US Government per deim rates | https://www.gsa.gov/travel/plan-book/per-diem-rates], and airfare costs should be justified. 
+
+2. The financd committee will review the costs and provide approval within 1 week of the request. 
+
+3.  Travel occurs
+
+4.  Preferable within two weeks of completing the trip, travel costs are submitted via the [NumFOCUS Rocket form | https://numfocus.org/rocket]
+
+5. NumFOCUS provides the information for approval to the Finance Committee.
+
+6. At least, one member of the finance committee provides approval for the payment.
+
+7. Payment is made to the requester for travel. 
+
+
+## Contractor Work
+
+1.  A request for contract work should be made via email to astropy-finance-committee@googlegroups.com.  
+
+2.  If the request is not coming from the coordination committee, the finance committee will seek approval for this contract.   
+
+3.  The terms of the contract work are negotiated (by FC? by NumFOCUS?)  
+
+4.  The contract is approved by 60% (?) of the finance committee.
+
+5.  The contract begins.  Hours worked are reported through the [NumFOCUS Rocket form | https://numfocus.org/rocket]. 
+
+6.  NumFOCUS provides the information for approval to the Finance Committee.
+
+7.   At least one member of the finance committee provides approval for the payment.  
+
+8.   Payment is made to the contractor.
+
+9.   Contracts should be reviewed yearly.   If necessary, renewals or extensions can be requested via email to astropy-finance-committee@googlegroups.com.
+
+
+## Any other requests for funding
+
+Any other requests for funding should be made via an email to astropy-finance-committee@googlegroups.com.   Please include your name, a description of the funding to be requested, and a budget. 
+
```

Comment:
The interim finance committee may need to revise this policy on an a case-by-case basis as needed and will make every effort to revise this policy as needed.